chainer/functions/connection/dilated_convolution_2d.py

```diff
@@ -61,6 +70,9 @@ def check_type_forward(self, in_types):
             )
 
     def forward_cpu(self, inputs):
+        if getattr(self, '_func', False):
```

Comment:
Knowing that we will work on merging tasks (2) right after this PR, this part is unused in this PR as far as I could see. So, to avoid confusion, it might be better to remove this part in this PR. How do you think about this? The same thing holds for `backward_cpu`.